bin/cozy-light

```diff
@@ -59,7 +69,7 @@ program
   .action(actions.enable);
 
 program
-  .command('install-distro [distro]')
+  .command('install-distro <distro>')
```

Comment:
Need to PR tj/commander to update the readme and makes it more clear, i m having same difficulty.